lib/blpapi-wrapper.ts

```diff
@@ -287,7 +287,7 @@ export class Session extends events.EventEmitter {
         return Promise.all(_.map(subscriptionsToServices(subscriptions), (uri): Promise<void> => {
             return this.openService(uri);
         })).then((): void => {
-            log('Subscribing to: ' + JSON.stringify(subscriptions));
+            //log('Subscribing to: ' + JSON.stringify(subscriptions));
```

Comment:
I came across this interesting issue and welcome any input. Basically there's an error `Converting circular structure to JSON` thrown by calling `JSON.stringify(subscriptions)`

The subscriptions object looks like this:
```
[{ domain:
     { domain: null,
       _events: [Object],
       _maxListeners: 10,
       members: [Object] },
    _events: { data: [Function] },
    _maxListeners: 10,
    correlationId: 1,
    security: 'GOOG US Equity',
    fields: [ 'LAST_PRICE' ],
    options: undefined }]
```
The Subscription object is defined like this in typescript:
```javascript
class Subscription extends events.EventEmitter  {
    correlationId: number;
    security: string;
    fields: string[];
    options: any;
    // constructor, etc...
}
```
So the circular reference comes from the nested `domain` properties. I did some research and here's what [Node](http://nodejs.org/api/domain.html#domain_class_domain) docs said:

![screen shot 2015-01-26 at 4 39 37 pm](https://cloud.githubusercontent.com/assets/3301112/5912358/ec0d7fe6-a587-11e4-8bab-8454dc121ec4.png)
![screen shot 2015-01-26 at 6 21 08 pm](https://cloud.githubusercontent.com/assets/3301112/5912376/243975aa-a588-11e4-9a35-6b2117adbcd1.png)

Resitfy uses this domain functionality to handle errors, and because of how typescript translate class inheritance(it seems to me that it will always elevate base "class" property up to the derived class), `domain` becomes an own property of 'subscription`, thus making the `JSON.stringify` error.

Any thoughts?